certified-connectors/timeghost/apiProperties.json

```diff
@@ -84,7 +84,7 @@
         }
       }
     ],
-    "publisher": "Koellisch Gesellschaft für Prozessmanagement mbH",

```

Comment:
@mattybeard Please let us know why you are adding another connector's update in your PR